lib/fqn/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/opentdf/platform/lib/fqn
+
+go 1.23
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/testify v1.10.0 // indirect
```

Comment:
Shouldn't this be direct?

Reply:
Yes - good eye. I forgot to `go mod tidy` this new module. Pushed up a new commit for that.